### DISCUSSION TITLE
Could you please add Jenius (Bank BTPN) to the converter?

### DISCUSSION THREAD

**Discussion by Ragenrave** [sarahsau/indoynab#3] [Adding a bank: request]
Hi,

First of all, I want to say thank you for making this web app. I've used YNAB since 2013 but I stopped using it around 2016 because I was too lazy to input each transaction manually. Now I started using it again mainly because of your converter. Thanks again!

A little request, Could you please add Jenius (Bank BTPN) to the list? Here is the sample of statements from that bank:
[e-Card Bank Statement](https://drive.google.com/file/d/1WDrmv_AKVHYnO_cdl3YiaoB8ueCFxYzp/view?usp=sharing)
[Main Account Bank Statement](https://drive.google.com/file/d/1gFu4twvCKCrXF-NokstEFidL-A7m3Uet/view?usp=sharing)

So there are two statements as far as I know. The first one is for the eCard, their virtual debit card that people use to shop online. The second is for the main bank account. The eCard gets its fund from the main account via a top-up. Really appreciate it if you can add it to the app, It could definitely change my financial life and other Jenius & YNAB users.

**Comment by sarahsau** [comment]
Hi Ragen, I’m glad you are using indoynab and came back to YNAB! 

Sure, let me see what I can do. Will try to take a stab on it on the weekend, and let you know once it’s done. 

Have a good day!

**Comment by sarahsau** [comment]
Hi @Ragenrave, I finished adding the Jenius converter. Could you confirm that it is working properly?

**Reply by Ragenrave** [reply]
Hi @sarahsau, Sorry for the late reply and thank you for responding to my request!

The Jenius online banking web app is still in constant maintenance in the past few weeks so I couldn't download the statement. Then I realized just recently they have the download option on their mobile app as well *facepalm*. Anyway, I've tried it today and unfortunately, it doesn't work. The CSV. file just shows a blank sheet with column titles on top (Date, Payee Memo, and amount). 

I want to help but I'm probably the least techie person in this Github. So, quick question, can I hire someone else from Fiverr, Upwork, etc. to help you create this converter for Jenius? I may need another converter for other banks as well since they're going to migrating my BNI Syariah account to the new Bank Syariah Indonesia but I don't want to bother you more. 

Let me know if that's okay and perhaps if you can give a little guide on what should I say to the freelancer on what kind of job that they're going to work on that would be great  

Have a good day and stay healthy!

**Reply by sarahsau** [reply]
Hei @Ragenrave, no worries! It's my pleasure to create new formats, it's my way to relax and have fun :) I think the error is because I was trying to guess the data model for the sanitized transaction lists. 

Could you email me the original statement at itsme [at] sarahsausan [dot] com? You can blank the account number and your real name (if you prefer), but I'd need the transaction lists untouched. Also if you have the BSI statement already, you can send along as well.

**Reply by Ragenrave** [reply]
Hi @sarahsau

Again, sorry for the late reply, but I finally finished my BSI migration. I'm going to send both statements in a bit. Thank you so much!
